cit3.0-web/src/components/Page/PowerBi/CriteriaSearch.js

```diff
@@ -7,7 +7,7 @@ import Config from "../../../Config";
 import "./PowerBi.css";
 
 export default function CriteriaSearch() {
-  const [report, setReport] = useState();
+  const [, setReport] = useState();
```

Comment:
is this correct syntax to have the , in there?

Reply:
if we just need to setReport, you can skip items while destructuring by using a comma without an variable or constant name.